src/Vacuum/Vacuum.jl

```diff
@@ -0,0 +1,74 @@
+module VacuumMod
+
+export mscvac, set_dcon_params
+
+
+const libdir = joinpath(@__DIR__, "..", "..", "deps")
+const libvac = joinpath(libdir, "libvac")
+
+function set_dcon_params(mthin::Int32, lmin::Int32, lmax::Int32, nnin::Int32,
```

Comment:
Users are gonna want to pass native system integers(i.e., `Int`, which is `Int64` on most systems), meaning just calling it with `mthin, lmin, lmax, nnin = 4, 1, 4, 2` instead of forcing the user to wrap in `Int32` like you do in runtests.jl. I would either expand the type to `::Integer` here and then wrap each in `Int32(mthin)`, etc. in the function. Alternatively you could have 
```
function set_dcon_params(mthin::Integer, lmin::Integer, lmax::Integer, nnin::Integer, qa1in::Float64, xin::Vector{Float64}, zin::Vector{Float64}, deltain::Vector{Float64})
    return set_dcon_params(Int32(mthin), Int32(lmin), Int32(lmax), Int32(nnin), qa1in, xin, zin, deltain)
end
```